yjit/src/stats.rs

```diff
@@ -237,6 +237,8 @@ make_counters! {
     send_send_chain_not_string_or_sym,
     send_send_getter,
     send_send_builtin,
+    send_is_a_known_class,
+    send_instance_of_known_class,
```

Comment:
These counter track when the class doesn't match what we expect right?

Maybe we could call them `_class_mismatch` instead of `_known_class` ?

Everything else looks great 👌